bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -192,13 +193,26 @@ class App extends Component {
     );
   }
 
+  renderActivityCheck() {
+    const { User } = this.props;
```

Comment:
Is it a good idea to pass the whole `User` in props? Looks like you only need `inactivityCheck, responseDelay`

Reply:
We receive the user from here: https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/startup/client/base.jsx#L200, I'll need to check if this `User` is used somewhere else.